File: Java/src/Lab_1/Calculator.java
package Lab_1;

import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("________________________");
        System.out.println("        Calculator      ");
        System.out.println("________________________");
        System.out.print("Enter Two Numbers: ");
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        System.out.println("Operations:");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");
        System.out.print("Your Choice: ");
        int choice = scanner.nextInt();
        switch (choice) {
            case 1:
                System.out.printf("%d + %d = %d", a, b, a + b);
                break;
            case 2:
                System.out.printf("%d - %d = %d", a, b, a - b);
                break;
            case 3:
                System.out.printf("%d * %d = %d", a, b, a * b);
                break;
            case 4:
                System.out.printf("%d / %d = %d", a, b, (float) a / b);
                break;
            default:
            System.out.println("INVALID CHOICE.");
        }
    }
}

File: Dotnet (C#)/Lab 1/calculator.cs
// WAP to input two numbers and perform addition, substraction, multiplication and division

using System;

class Calculator {
    static void Main(string[] args) {
        Console.WriteLine("Enter Two Numbers: ");
        int numOne = int.Parse(Console.ReadLine());
        int numTwo = int.Parse(Console.ReadLine());
        Console.WriteLine("1. Addition");
        Console.WriteLine("2. Substraction");
        Console.WriteLine("3. Multiplication");
        Console.WriteLine("4. Division");
        Console.Write("Enter your choice: ");
        int choice = int.Parse(Console.ReadLine());
        switch(choice) {
            case 1:
                Console.WriteLine("The sum is: " + (numOne + numTwo));
                break;
            case 2:
                Console.WriteLine("The substraction is: " + (numOne - numTwo));
                break;
            case 3:
                Console.WriteLine("The Multiplication is: " + (numOne * numTwo));
                break;
            case 4:
                double numOneD = (double)numOne;
                double numTwoD = (double)numTwo;
                Console.WriteLine("The division is: " + (numOneD / numTwoD));
                break;
            default:
                Console.WriteLine("Invalid Choice. Exiting..");
                break;
        }
        return;
    }
}